charts/frontend/values.yaml

```diff
@@ -18,11 +18,11 @@ appImage:
   tag: latest
 
 nginxImage:
-  repository: nginx
+  repository: nginxinc/nginx-unprivileged
   pullPolicy: Always
-  tag: stable
+  tag: 1.21.3
```

Comment:
We'd need a way to keep this up to date. nginx-unprivileged maintains a [`stable` tag](https://hub.docker.com/r/nginxinc/nginx-unprivileged/tags) like the other nginx image, so I'd recommend sticking with that for now. (If you do set an immutable tag (like 1.21.3) then you should remove `pullPolicy: Always` - but I wouldn't do that without an update mechanism in place because it'd be bad if this got forgotten and made it through to production with a pinned version and no automatic updates).

Reply:
Cool. The reason I chose the version above was support for `linux/amd64` which is what I _assume_ we require. Looking again now at _stable_ I must have missed this support as it's listed for that flag also.